### ISSUE TITLE
Maybe tag a new -devel version?

### ISSUE THREAD

**Issue by r3pek** [akinomyoga/ble.sh#242]
It's been almost 2 years since a tagged version was made (428 commits and 145 files changed). Maybe a new release can be made?

**Comment by akinomyoga** [comment]
Yes, I'm currently planning to tag a new devel version, but I'm just busy.

**Comment by Knusper** [comment]
Hi, I really also would like to have a "real" release rather than always being on the bleeding edge here.... I started using ble basically everyday, and I hope always that a `ble-update` does not braeak things... if we could have releases, this would make things easier (also going back and forth between versions, in case something breaks).

**Comment by akinomyoga** [comment]
I tagged the final version of 0.4.0-devel3. Now the master is 0.4.0-devel4.
